app/views/layouts/application.html.erb

```diff
@@ -15,7 +15,7 @@
         <div class="border-b border-white/25 p-4 text-4xl font-thin">Railicious</div>
         <nav class="flex grow flex-col justify-between gap-4">
           <ul class="flex flex-col items-stretch justify-between">
-            <li><a href="#" class="block px-4 py-2 hover:bg-white/10">Public Recipes</a></li>
+            <li><%= link_to "Public Recipes", root_path, class:"block px-4 py-2 hover:bg-white/10" %></li>
```

Comment:
It is true that we show same page if we visit root, but we should have a dedicate path for it. Better to link it to public_recipes page. It is one of the requirements to have such path.